tests/model_forms/tests.py

```diff
@@ -2098,7 +2098,8 @@ def test_model_multiple_choice_field(self):
             [self.c1, self.c2],
         )
         with self.assertRaises(ValidationError):
-            f.clean(["100"])
+            max_pk = max(Category.objects.values_list("pk", flat=True))
+            f.clean([f"{max_pk + 1}"])
```

Comment:
Can we use "0" instead?
```suggestion
            f.clean(["0"])
```

Reply:
Hi, sorry for replying back this late. 

I liked this suggestion, but after checking it out in the debugger I'm afraid this might not work. 

If we supply a pk of "0" we raise on line 1621 i.s.o. 1630 of `forms/models.py`. 

![image](https://user-images.githubusercontent.com/16463246/163032484-28bf7891-f04f-4cbd-a83a-9ac243d87a22.png)

Hence we would be testing another code path, which we also already test with

         with self.assertRaises(ValidationError):
              f.clean(["fail"])
 

----

However I can understand that the solution we provided might be undesirable by doing an extra computation.  Another possibility would be a significantly larger integer. 

As we are a first time contributor I'm also not very aware of the conventions in the Django test code base. 

Looking forward for your take on all this.

Reply:
Whoops, scratch that

Reply:
Clever idea indeed. That should work and save us the computation in the test.